automerge/src/error.rs

```diff
@@ -7,6 +7,8 @@ use thiserror::Error;
 pub enum AutomergeError {
     #[error("invalid opid format `{0}`")]
     InvalidOpId(String),
+    #[error("invalid actor `{0}`")]
+    InvalidActor(String),
```

Comment:
Is this necessary to have in the public API? It would be nice if we could leave it out as it exposes the internal structure of the OpIDs, which I'm really keen to avoid.

Reply:
The problem was the upwelling team was feeding a valid opid into the system (but it was from the wrong document) and they were seeing "invalid opid format" - i needed a different error for when its a valid format but for an actor not in the document.   Suggestions?

Reply:
Ah interesting. i think two things

1. Just having "opid not part of this document" would be as informative right? (and possibly preferable because a user who doesnt know much about actors would atill have a chance of knowing what to do)
2. We should add aome logs (via `tracing` for debugging these conditions when its a programming wrror we have made.